olive/common/hf/peft.py

```diff
@@ -28,15 +28,13 @@ def make_export_compatible_peft(model: torch.nn.Module, merge_weights: bool = Fa
         - Rescale the lora_B weights with scaling factor, change scaling factor to 1 (int) so that it doesn't appear
           in the exported model (only works for torchscript export)
     """
-    if not is_peft_model(model):
-        return model
-
     if merge_weights:
         return model.merge_and_unload()
 
     # if pytorch_model is PeftModel, we need to get the base model
     # otherwise, the model forward has signature (*args, **kwargs) and torch.onnx.export ignores the dummy_inputs
-    model = model.get_base_model()
+    if is_peft_model(model):
+        model = model.get_base_model()
 
     from peft.tuners.lora import LoraLayer
```

Comment:
also, this import needs to be in a try block. we can just return if import fails since it guarantees the model doesn't have loralayers

Reply:
If we just return here, this will never be triggered cause peft is not required for conversion. Is this expected?

Reply:
yes, that is expected. if we cannot import peft then the model cannot have the peft lora layers.